README.md

```diff
@@ -58,9 +58,25 @@ Try to write PR titles in the conventional commit format, and squash and merge w
 
 ### Working locally with DCR
 
-To use the bundle locally with DCR, run `COMMERCIAL_BUNDLE_URL=http://localhost:3031/graun.standalone.commercial.js PORT=3030 make dev` in the DCR directory.
+1.  To point DCR to the local commercial bundle, in the `dotcom-rendering/dotcom-rendering` directory run:
 
-DCR will then use the local bundle instead of the one from PROD/CODE.
+    `COMMERCIAL_BUNDLE_URL=http://localhost:3031/graun.standalone.commercial.js PORT=3030 make dev`
+
+    This will override `commercialBundleUrl` passed via the page config from PROD/CODE.
+
+1. In another terminal start the commercial dev server to serve the local bundle:
+
+    `yarn serve`
+
+## Testing locally with DCR
+
+Follow the steps above to run DCR against the local bundle.
+
+To run the Playwright e2e tests:
+
+`yarn playwright:run` will run the tests on the command line
```

Comment:
Nice update! Could we update the commands on line 28 as well?

Reply:
Oh good spot thanks!